### ISSUE TITLE
Implement AdhocBrowserPool support for activity_config basic params

### ISSUE THREAD

**Issue by wzin** [EndPointCorp/lg_ros_nodes#164]
@wzin commented on [Mon Feb 29 2016](https://github.com/EndPointCorp/ros_cms/issues/303)

tl;dr - we're going to send dynamic/optional/additional data or configuration inside `activity_config` in USCS/director messages. AdhocBrowserPool is responsible for managing AdhocBrowsers so it needs to interpret this attribute correctly. It will typically contain extensions list, user agent etc.

Requirements:
- parse activity_config only if it's there
- make support for retrieving following configuration parameters in `activity_config`:
 - list of extensions (`path`, `name`, `extension_data_source`)
 - user agent
 - binary
 - audio enabled (true|false) (`--mute-audio` flag)
 - custom_flags e.g. `--disable-out-of-process-pac` (obsessively check whether they match regexp and someone is not trying to do `;bash -c rm -fr  /`) or `--enable-arc` or `--enable-nacl`

**Comment by wzin** [comment]
https://github.com/EndPointCorp/lg_ros_nodes/commits/issue/164_activity_config
